conf/routes

```diff
@@ -27,7 +27,8 @@ PATCH   /api/v1/admin/updatePicture                         controllers.Restaura
 GET     /api/v1/getAllRestaurants                           controllers.RestaurantController.getAllRestaurants
 GET     /api/v1/getNumberOfRestaurants                      controllers.RestaurantController.getNumberOfRestaurants
 GET     /api/v1/getNearbyRestaurants/:latitude/:longitude   controllers.RestaurantController.getNearbyRestaurants(latitude: Float,longitude: Float)
-GET     /api/v1/getPopularRestaurants                       controllers.RestaurantController.getPopularRestaurants
+
+GET     /api/v1/getPopularRestaurants/:id                   controllers.RestaurantController.getPopularRestaurants(id: String)
```

Comment:
Passing the `User ID` through a `Public API` ? So, I (as a User) can see the habits of any other user so long as I know their ID? Get the currently logged in user instead, using the Cache API in the controller.